libvast/vast/system/actor_factory.hpp

```diff
@@ -0,0 +1,75 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2018 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include "vast/fwd.hpp"
+
+#include "vast/system/actors.hpp"
+
+namespace vast::system {
+
+template <class Type>
+struct factory_traits {
```

Comment:
This whole file should likely not be in here.

Reply:
Yes, I should be more careful with `git commit -a`.